docker/CMakeLists.txt

```diff
@@ -13,6 +13,13 @@ add_custom_target(DockerBuildDevelopment
   VERBATIM
 )
 
+add_custom_target(DockerRunDevelopment
+        docker run --rm -it --workdir /home/ubuntu/parflow/src --env=DISPLAY --name=parflow-development -e LANG=en_US.utf8 -v ${CMAKE_CURRENT_SOURCE_DIR}/..:/home/ubuntu/parflow/src parflow-development
+        COMMENT
+        "Start docker image 'parflow-development' mounted to source directory to allow live rebuilding"
+        VERBATIM
+        )
+
```

Comment:
This creates a make command "make DockerRunDevelopment" that when run starts the dev container and mounts it to your code so that you can make edits to your local code and update the container by running the "rebuild" command inside of it